index.d.ts

```diff
@@ -14,6 +14,8 @@ export declare class ChatMessage {
     displayWarning?: boolean
   )
 
+  public readonly json: string
```

Comment:
```suggestion
  public readonly json: any
```
it's an object, not a string

Reply:
wuuops yes you are right